docs/http_api.rst

```diff
@@ -0,0 +1,1099 @@
+
+    Remove an existing user.
+
+    :status 204: Successfully deleted the user.
+
+    **Example request**:
+
+    .. sourcecode:: http
+
+        DELETE /users/1 HTTP/1.1
+
+    **Example response**:
+
+    .. sourcecode:: http
+
+        HTTP/1.1 204 No Content
+
+.. http:put:: /users/(int:user_id)/team
+
+    Place a user in a team
+
+    :<json int team_id: The ID of the team to place the user in
+
+    :>json int id: The ID for the user.
+    :>json str url: The URL for the user.
+    :>json str first_name: The (optional) first name of the user.
+    :>json str last_name: The (optional) last name of the user.
+    :>json str email: The email address of the user.
+    :>json bool admin: True if the user is an admin user.
+    :>json list teams: A list of all the teams a user is a member of.
+    :>json list organizations:
+        A list of all the organizations the user is a member of.
+
+    :status 200: Successfully added the user to the team.
+
+    **Example request**:
+
+    .. sourcecode:: http
+
+        PUT /users/1/team HTTP/1.1
+        Content-Type: application/json
+
+        {
+            "team_id": 2
+        }
+
+    **Example response**:
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Content-Type: application/json
+
+        {
+            "id": 1,
+            "url": "https://example.org/users/1",
+            "first_name": "Jon",
+            "last_name": "Snow",
+            "email": "jonsnow@castleblack.net",
+            "admin": false,
+            "teams": [
+                {
+                    "id": 2,
+                    "url": "https://example.org/teams/2"
+                }
+            ],
+            "organizations": []
+        }
+
+.. http:delete:: /users/(int:user_id)/teams/(int:team_id)
+
+    Remove the user from the specified team.
+
+    :>json int id: The ID for the user.
+    :>json str url: The URL for the user.
+    :>json str first_name: The (optional) first name of the user.
+    :>json str last_name: The (optional) last name of the user.
+    :>json str email: The email address of the user.
+    :>json bool admin: True if the user is an admin user.
+    :>json list teams: A list of all the teams a user is a member of.
+    :>json list organizations:
+        A list of all the organizations the user is a member of.
+
+    :status 200: Successfully removed the user from the team.
+
+    **Example request**:
+
+    .. sourcecode:: http
+
+        DELETE /users/1/teams/2 HTTP/1.1
+
+    **Example response**:
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Content-Type: application/json
+
+        {
+            "id": 1,
+            "url": "https://example.org/users/1",
+            "first_name": "Jon",
+            "last_name": "Snow",
+            "email": "jonsnow@castleblack.net",
+            "admin": false,
+            "teams": [],
+            "organizations": []
+        }
+
+.. http:put:: /users/(int:user_id)/organization
```

Comment:
Okay sure. What about adding and removing users from teams? Should that be done on the user endpoints or the team endpoints?